library/std/src/primitive_docs.rs

```diff
@@ -233,7 +233,10 @@ mod prim_bool {}
 ///
 /// ```
 /// #![feature(never_type)]
-/// # use std::fmt::{self, Debug};
+/// # use std::fmt;
+/// # trait Debug {
+/// #     fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result;
+/// # }
```

Comment:
Can you post the error message you got in the commit message? Or possibly just pick a different trait to implement :P

Reply:
You mean put the error message in the commit message? Or post it here? I can do either, I just wasn't sure what you meant.

The error message is in the CI logs for 7e2548f by the way.

Reply:
Put the error in the commit message.
```
 error[E0119]: conflicting implementations of trait `std::fmt::Debug` for type `!`:
 --> src/primitive_docs.rs:236:1
  |
6 | impl Debug for ! {
  | ^^^^^^^^^^^^^^^^
  |
  = note: conflicting implementation in crate `core`:
          - impl std::fmt::Debug for !;

error[E0117]: only traits defined in the current crate can be implemented for arbitrary types
 --> src/primitive_docs.rs:236:1
  |
6 | impl Debug for ! {
  | ^^^^^^^^^^^^^^^-
  | |              |
  | |              `!` is not defined in the current crate
  | impl doesn't use only types from inside the current crate
  |
  = note: define and implement a trait or new type instead
```
Hmm, seems like this will pop up for any trait from core you try to implement. The dummy `Debug` seems fine then.

Reply:
Okay, added the errors to the commit message (now e13a701).